tools/build-debian-copyright

```diff
@@ -94,20 +97,25 @@ for directory, subdirs, files in os.walk('dist'):
 
         for comment in comments:
             licenses = find_patterns(license_patterns, comment)
-            if len(licenses) != 1:
-                raise SystemError('Can not determine license of:\n%s' % comment)
-            if licenses[0].lower() not in license_ids:
-                raise KeyError('License {licenses[0]} not found in {template_file}')
+            if not licenses:
+                raise SystemError('Can not determine licenses of:\n%s' % comment)
+            for license in licenses:
+                if license.lower() not in license_ids:
+                    raise KeyError('License {license} not found in {template_file}')
+
+            # All webpacks also contain our own code
+            licenses.add("LGPL-2.1-or-later")
+
+            dist_licenses.setdefault(f'{directory}/*', set()).update(licenses)
 
-            copyrights = '\n '.join(find_patterns(copyright_patterns, comment))
+            copyrights = find_patterns(copyright_patterns, comment)
             if not copyrights:
                 raise SystemError('Did not find any copyrights in:\n%s' % comment)
 
-            paragraph = f'Copyright: {copyrights}\nLicense: {licenses[0]}'
+            # All webpacks also contain our own code
+            copyrights.add("Copyright (C) 2013 - 2021 Red Hat, Inc.")
```

Comment:
I don't like the hardcoded "2021". To be completely precise we could iterate over `git grep 'Copyright.*Red Hat'`,  but I suppose for simplicity we could just take the current year?

Reply:
Replaced with current year.

![mediocrates](https://user-images.githubusercontent.com/200109/126902976-4bde00c1-22ab-42f8-8bb0-fb3c3a3e0eca.jpg)